introduction.md

```diff
@@ -1,2 +1,12 @@
 Hi, this is an introduction. I am FileNotFound and i am a computer science student in germany. Currently I am writing my master what includes using GitHub, so i take the corresponding Codeschool course.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?